### PR TITLE
Make the Snacktory testcases agnostic of host machine timezone

### PR DESCRIPTION
**Pull request by abhishekvm**
- When timezone is present in article date itself, parsing will happen correctly.
- If the timezone is not present in article date we should assume default timezone `UTC` instead of host machine timezone. It was taking machine timezone which was problematic.
- All the dates mentioned in testcases' asserts should either use the same timezone which is default timezone used for extraction or provided with timezone explicitly.
- Verified that all the testcases are running well by setting my machine clock to different timezones.

### CODE REVIEW

File: src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java [hunk 1]
```diff
@@ -1763,24 +1764,18 @@ private Date parseDate(String dateStr) {
             "dd.MM.yy", // 22.09.16
         };
 
+        Date date = null;
         try {
-            if(DEBUG_DATE_EXTRACTION){
-                System.out.println("BEFORE clean: dateStr="+dateStr+"|");
+            logger.debug("BEFORE clean: dateStr="+dateString+"|");
```
Comment (skyshard) [inline src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java]: change dateStr to dateString to be consistent

File: src/main/java/de/jetwick/snacktory/utils/Constants.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package de.jetwick.snacktory.utils;
+
+/**
+ * All the constants should go in this util class
+ *
+ * @author Abhishek Mulay
+ */
+final public class Constants {
+
+    public static final String DEFAULT_TIMEZONE = "UTC";
```
Comment (skyshard) [inline src/main/java/de/jetwick/snacktory/utils/Constants.java]: would this be better in an application properties / yml type file?

**Comment by abhishekvm** [comment]
@skyshard @andresp99999 Can you merge this if there is no more review comments ?

File: src/main/java/de/jetwick/snacktory/utils/Constants.java (hunk 2 above)
Comment (abhishekvm) [reply src/main/java/de/jetwick/snacktory/utils/Constants.java]: Introduced new Configuration class instead to load the config from yaml files.

**Pull request merged**